com.github.giacomogroppi.writernote-qt.json

```
@@ -0,0 +1,78 @@
+{
+  "app-id": "com.github.giacomogroppi.writernote-qt",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.2",
+  "sdk": "org.kde.Sdk",
+  "command": "writernote",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--share=ipc"
+  ],
+  "cleanup": [
+    "/lib/pkgconfig",
+    "/share/man",
+    "*.la"
+  ],
+  "modules": [
+    {
+      "name": "Libzip",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DCMAKE_BUILD_TYPE=Release",
+        "-DCMAKE_INSTALL_LIBDIR=/app/lib",
+        "-DCMAKE_INSTALL_INCLUDEDIR=7app/include"
```

```suggestion
        "-DCMAKE_BUILD_TYPE=Release"
```

You should need this. Also there is a typo int he `INCLUDEDIR`